articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+
+## Shares Tokens (sTokens)
+
+[sTokens](https://silopedia.silo.finance/deposit/shares-tokens-stokens) are ERC-20 tokens that represent a claim on the deposited asset. For instance, in return for depositing LINK into Silo, the lender receives sLINK. _sTokens_ also accrue interest, depending on the interest rate per token.
+
+When depositing tokens, Silo provides an innovative feature that allows lenders to choose whether their deposits can be borrowed by others, or whether they want to “protect” their deposits, which disables others from borrowing said assets. However, in return for protecting the token, the lender forgoes earning an interest rate. _sTokens_ represent borrowable assets, and _spTokens_ represent protected deposits. It’s also possible to do both for the same asset (e.g. 40% of the token is borrowable and 60% is protected). _spTokens_ can still be used as collateral to borrow other tokens.
+
+For bridge assets (ETH or XAI), the naming convention is slightly different, i.e._ sBridge-ABC_, whereby ABC represents the name of the silo that ETH or XAI is deposited into. For instance, when depositing the bridge asset ETH into the LINK silo, the lender in return receives _sETH-LINK_ or _spETH-LINK_.
+
+
+## Debt Tokens (dTokens)
+
+Conversely, [dTokens](https://silopedia.silo.finance/borrow/debt-tokens-dtokens) are ERC-20 tokens representing a borrowed asset. They are minted when an asset is borrowed and burned when the loan is paid back. Unlike _sTokens_, _dTokens_ cannot be transferred out of a wallet. For bridge assets, the borrowed token is represented again as _dBridge-ABC_ (e.g. dETH-UNI).
+
+
+## Conclusion
+
+In summary, Silo introduces a few innovations and a new money market primitive aiming to make lending markets less risky, while simultaneously improving market access for long-tail assets. The ability to protect tokens from borrowers can be a very interesting feature for other DAOs to deploy their tokens and borrow tokens against them.
+
+The risk reduction, however, has one trade-off with the current design: Borrowing at Silo can result in up to six transactions. First, &lt;allow deposit> and &lt;deposit> of the collateral, then &lt;borrow> the bridge asset, then again &lt;allow deposit> and &lt;deposit> the bridge asset, before finally the targeted asset can be &lt;borrowed>. This can be quite costly on the Ethereum mainnet and might be a blocker for smaller accounts to use Silo as intended. The team is working on new features to improve this, but it will take some time.
+
+
+# The XAI Stablecoin
+
+The Silo team aims to issue a new over-collateralized stablecoin with a soft peg to the US Dollar called XAI. The main use case for XAI is to serve as a second bridge token alongside ETH. Thus, enabling users to borrow XAI to bridge from token A to token B.
```

Comment:
My understanding is XAI will be an alternative borrowable asset in a particular silo. For instance, the USDC silo would allow to deposit USDC and borrow either ETH or XAI. If that's not true, maybe make that clear. 

Also, the language they use is that XAI "will be" a bridged asset. Is this function not yet live, and if not, is there a timeline for expected utility?

Reply:
Yes XAI will start as a borrowable asset in just two Silos, ETH and USDC. After a certain period (I assume they want to see how things are going), they will start to add XAI to more silos, so that it can become a bridge asset for more assets. There is a specific executive proposal required to enable new bridge assets. I hope we made this clear in the following sections. Lmk if not, happy to further adjust this part too.